src/Apps/CollectorProfile/Routes/Saves2/__tests__/CollectorProfileSaves2Route.jest.tsx

```diff
@@ -47,7 +47,7 @@ describe("CollectorProfileSaves2Route", () => {
       }),
     })
 
-    expect(screen.getByText("Saved Artworks")).toBeInTheDocument()
+    expect(screen.getAllByText("Saved Artworks").length).toBe(2)
```

Comment:
**suggestion**: rename `Saved Artworks` to `Saved Artwork` [here](https://github.com/artsy/force/blob/d994970462165af2ec918ea437e25e5af2693fb4/src/Apps/CollectorProfile/Routes/Saves2/__tests__/CollectorProfileSaves2Route.jest.tsx#L135) and leave the old test implementation